bodo/pandas/frame.py

```diff
@@ -24,6 +26,18 @@ class BodoDataFrame(pd.DataFrame, BodoLazyWrapper):
     # use it directly when available.
     _head_df: pd.DataFrame | None = None
 
+    def __init__(
+        self, data=None, index=None, columns=None, dtype=None, copy=None, plan=None
+    ):
+        self.__dict__["plan"] = plan
```

Comment:
Using __dict__ doesn't seem necessary since "plan" is not a dynamic value?

Reply:
Same comment as above.

Reply:
Same.

Reply:
Fixed using _internal_names and _internal_names_set from Pandas.